web/src/settings_exports.ts

```diff
@@ -297,6 +316,99 @@ export function refresh_allow_private_data_export_banner(): void {
     }
 }
 
+function maybe_show_notes_about_unusable_users_if_exported(export_type: number): void {
+    // Show warnings if there are users whose accounts will be inaccessible
+    // once exported. A user account will be inaccessible if they:
+    //  - Don’t consent to their private data being exported and “Standard
+    //    export” is selected.
+    //  - Have set their email visibility to “nobody” and “Public export”
+    //    is selected.
+    let unusable_user_ids: number[] = [];
+    if (export_type === settings_config.export_type_values.export_full_with_consent.value) {
+        const non_consenting_users = get_export_consents_having_consent_value(false);
+        unusable_user_ids = non_consenting_users.map((user) => user.user_id);
+    } else if (export_type === settings_config.export_type_values.export_public.value) {
+        const users_with_inaccessible_email = get_export_consents_having_email_visibility_value(
+            settings_config.email_address_visibility_values.nobody.code,
+        );
+        unusable_user_ids = users_with_inaccessible_email.map((user) => user.user_id);
+    } else {
+        return;
+    }
+
+    if (unusable_user_ids.length === 0) {
+        return;
+    }
+
+    // Show the number of users whose accounts will be inaccessible.
+    $("#unusable_user_accounts_note").html(
+        $t_html(
+            {
+                defaultMessage: `
+                If you import the data into a new Zulip organization,
+                {count, plural, one {# user} other {# users}} will be
+                unable to log in. Their email {count, plural, one
+                {address} other {addresses}} will not be included in
+                the export because you don't have permission to
+                <z-link>access</z-link> or export their {count, plural,
+                one {email} other {emails}}.
+                `,
```

Comment:
This has so many plurals in it that it'll prove unrealistic to translate. I think we should redo the string. Something like:

> `You do not have permission to <z-link>access</z-link> the email addresses of {count, plural, one {# user} other {# users}} in this organization. Affected users will be exported with a fake email address, and manual action will be required to associate a new email address with their accounts for future authentication.`


But probably needs some fiddling. @alya thoughts?

Reply:
Yeah, hm, that version has an "addresses" that will be wrong sometimes. How about this?

> You don't have permission to <z-link>access</z-link> the email {count, plural, one
{address} other {addresses}} of {count, plural, one {# user} other {# users}}. Accounts for users whose emails you can't access will be created with placeholder email addresses. A placeholder email must be updated by an administrator for the user to be able to log in.